src/layout/Body.js

```diff
@@ -0,0 +1,105 @@
+import React, { Fragment, Component } from 'react';
+import {
+    Bullseye,
+    Card,
+    CardHeader,
+    CardBody,
+    CardFooter,
+    Split,
+    SplitItem,
+    Grid,
+    GridItem,
+    TextContent,
+    Text,
+    TextVariants
+} from '@patternfly/react-core';
+import { ArrowRightIcon, BinocularsIcon } from '@patternfly/react-icons';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+class Body extends Component {
+    onLaunchClick = (event, name, url, title) => {
+        event.preventDefault();
+        localStorage.setItem('cs-app', name);
+        localStorage.setItem('cs-app-title', title);
+        window.location.href = `${document.baseURI}platform/${url}`;
+    }
+
+    render() {
+        const { technologies } = this.props;
+        return (
+            <Fragment>
+                <div>
+                    <Bullseye>
+                        <TextContent>
+                            <Text component={ TextVariants.h1 }>
+                                Explore Cloud Technologies
+                            </Text>
+                        </TextContent>
+                    </Bullseye>
+                </div>
+                <Grid sm={ 12 } md={ 6 } lg={ 3 } gutter="md">
+                    { technologies.map(({ icon: Icon, iconProps, title, url, body, isPreview, name }, key) => (
+                        <GridItem key={ key }>
+                            <Card className="ins-c-application-info">
+                                <CardHeader><Icon size="xl" className="ins-c-icon__active" { ...iconProps } /></CardHeader>
```

Comment:
I don't know if I would put this in a `<CardHeader>`, I'd say stack it in the card body and then add a gutter.

Reply:
What's the reasoning behind this? It looks the same.

Reply:
Whoops. This was meant for the bottom "get documentation" "open a support case" etc. The icons are too close to the text

Reply:
Oooh, yeah I'll uodate that as well.